pkg/controller/storagecluster/controller_test.go

```diff
@@ -9164,6 +9164,23 @@ func TestShouldPreflightRun(t *testing.T) {
 	require.True(t, preflightShouldRun(cluster))
 	logrus.Infof("vshpere cloud w/PX >= 3.1, preflight will run")
 
+	// TestCase: Vsphere cloud provider with Install mode 'local'
+	logrus.Infof("check vsphere cloud w/local install mode...")
```

Comment:
nit: debug logs? remove?

Reply:
These are logs added to separate the test output.